tools/development/nnstreamerCodeGenCustomFilter.py

```diff
@@ -128,12 +128,15 @@ def getinput(text):
   in_info->num_tensors = 1; /** @todo MODIFY THIS! */
   in_info->info[0].name = NULL; /** Optional, default is null. Set new memory for tensor name string. */
   in_info->info[0].type = _NNS_UINT8; /** @todo MODIFY THIS! */
+
+  /**
+   * Check max dimension (NNS_TENSOR_RANK_LIMIT) and
+   * fill 0 to uninitialized dimension values.
```

Comment:
Who has the responsibility to do things described in this comment?

Reply:
If you mean the todo tag, it is developer to set valid tensor metadata.

Reply:
Um, what I meant is ... the previous codes initialized remaining dimension slots to 1 (up to `NNS_TENSOR_RANK_LIMIT`).
Now, new codes set dimension values only up to [3]. So, I was confused because I first thought that this comment describes the below codes. 
But, you meant that the duty of filling values for remaining slots is now moved to developers, right? Then, it's fine.